SynthSeg/training_options.py

```diff
@@ -400,5 +414,7 @@ def get_non_path_string_options() -> List[str]:
             "prior_distributions",
             "wandb_log_freq",
             "compression_type",
-            "strategy"
+            "strategy", 
+            "find_last_checkpoint", 
+            "seed"
```

Comment:
```suggestion
            "strategy"
```
No need to add those, since they are not strings